internal/command/deploy/machines_deploymachinesapp.go

```diff
@@ -323,7 +323,10 @@ func (md *machineDeployment) updateUsingBlueGreenStrategy(ctx context.Context, u
 	bg := BlueGreenStrategy(md, updateEntries)
 	if err := bg.Deploy(ctx); err != nil {
 		fmt.Fprintf(md.io.ErrOut, "Deployment failed after error: %s\n", err)
-		return bg.Rollback(ctx, err)
+		if rollbackErr := bg.Rollback(ctx, err); rollbackErr != nil {
+			return rollbackErr
+		}
+		return err
```

Comment:
Do we still need the original error in the event that rollback failed?

Reply:
What do you mean?

Reply:
Since `err` is returned if deployment fails, but `rollbackErr` is returned if deployment failed _and_ rollback failed, I was curious if anything upstream depended on `err` being returned. But I see that the old code didn't return it at all, so I guess not!